SimPEG/potential_fields/gravity/simulation.py

```diff
@@ -116,16 +116,20 @@ def evaluate_integral(self, receiver_location, components):
                     g_c = [g_cx g_cy g_cz]
 
         """
-        tol1 = 1e-4
-        tol2 = 1e-10
+        tol1 = 1e-4  # Tolerance 1 for numerical stability over nodes and edges
+        tol2 = 1e-10  # Tolerance 2 for numerical stability over nodes and edges
```

Comment:
```suggestion
       
```
tol2 is not even used for gravity. We should just delete this one (I am ok keeping the other one named `tol1` for consistency with `magnetics`)